xbmc/cores/VideoPlayer/DVDCodecs/Video/VDPAU.cpp

```diff
@@ -579,6 +579,19 @@ bool CDecoder::Open(AVCodecContext* avctx, AVCodecContext* mainctx, const enum A
       mainctx->slice_flags = SLICE_FLAG_CODED_ORDER|SLICE_FLAG_ALLOW_FIELD;
       mainctx->hwaccel_context = &m_hwContext;
 
+      // update processInfo
+      std::list<EINTERLACEMETHOD> deintMethods;
```

Comment:
VDPAU needs to wait - on way to work - no compile testing possible. VAAPI updated accordingly. If you squash anyways, move it please and let jenkins verify :-)